guides/working-with-starcounter/working-in-visual-studio/README.md

```diff
@@ -0,0 +1,154 @@
+# Working in Visual Studio
+
+Developing Starcounter applications is straightforward and not tied to any certain development environment. Applications targeting .NET are normally built using Visual Studio, but would work using just `msbuild` or even the C# compiler too.
+
+In Visual Studio, applications can be built using a [Starcounter Visual Studio extension](https://marketplace.visualstudio.com/items?itemName=Starcounter.StarcounterforVisualStudio), or just using standard Visual Studio templates such as the C# "Console Application" project template.
+
+# Using the Visual Studio extension
+
+Starcounter provides a Visual Studio extension to simplify the development of Starcounter applications. It provide templates to build on and the ability to start applications from Visual Studio.
+
+## Templates
+
+There are currently two project templates and three item templates.
+
+**Project templates**:
+
+The project templates are used to scaffold projects that target Starcounter. When instantiated, the project will have some default Starcounter-specific *assembly references* set up and include some build hooks that will aid in building. For more information on the specifics, see section about [creating applications using standard templates](Create-a-Starcounter-application-using-standard-templates) later in this text.
+
```

Comment:
For internal links to work, you have to use all lowercase and start with a hash, like this: 
`[creating applications using standard templates](#create-a-starcounter-application-using-standard-templates)`